v2.2/recommended-production-settings.md

```diff
@@ -39,54 +39,113 @@ Term | Definition
 For added context about CockroachDB's fault tolerance and automated repair capabilities, see [this training](training/fault-tolerance-and-automated-repair.html).
 {{site.data.alerts.end}}
 
+
+## Operating Systems
+
+For information about operating system requirements, see the relevant Install page:
+
+- [Install CockroachDB on Mac](install-cockroachdb-mac.html)
+- [Install CockroachDB on Linux](install-cockroachdb-linux.html)
+- [Install CockroachDB on Windows](install-cockroachdb-windows.html)
+
 ## Hardware
 
 ### Basic hardware recommendations
 
-- Nodes should have sufficient CPU, RAM, network, and storage capacity to handle your workload. It's important to test and tune your hardware setup before deploying to production.
+Nodes should have sufficient CPU, RAM, network, and storage capacity to handle your workload. It's important to test and tune your hardware setup before deploying to production.
+
+#### CPU, memory, and storage recommendations
+
+- At a bare minimum, each node should have **2 GB of RAM and one entire core**.
+
+    More data, complex workloads, higher concurrency, and faster performance require additional resources; as a general rule of thumb, increase the number of cores and additional memory to match with a ratio of 4-8 GB of RAM per core, never exceeding a maximum of 16 GB. On larger socket systems, NUMA should be enabled.
```

Comment:
Good point, as it should read ' for hypervisor configurations which have more than one socket, Numa best practices should be followed for optimal memory performance.'  However, this can be more complex as if 2 numa nodes appear in an OS, which is possible on large containers on cloud platforms, we would need to test it.  What would be the best practice for 2 numa nodes in one container, 1 CR or 2 CR numa bind to each node.   This would need to be tested.